### DISCUSSION TITLE
can there be multiple In-Network Objects with the same field values

### DISCUSSION THREAD

**Discussion by nmayle** [CMSgov/price-transparency-guide#516] [Q&A] [answered]
Can there be multiple In-Network Objects with the same value for: negotiation_arrangement, name, billing_code_type, billing_code_type_version, billing_code_type_version and description?

Can be there be multiple records with all of those fields the same, but with different arrays of negotiated_rates?
Shouldn't those be combined?  Is there any requirement to combine these fields or can every negotiated_rate be in its own In-Network Object?

**Comment by shaselton-usds** [comment, marked as answer]
No requirements and there is the ability to duplicate the object with different `negotiated_rates` though I wouldn't advise it if you're serious about file size. This would be a basic optimization if you're producing the files with all the downside for file hosting and bandwidth costs with little downside to those consuming the files due to the streaming consumption that is expected.
